www/PoyntPlugin.js

```diff
@@ -12,6 +12,50 @@ var Poynt = {
 				"amount": amount, 
 				"referenceId": referenceId
 			}]);
+	},
+	launchBilling: function (paymentid, successCallback, errorCallback){
+		cordova.exec(successCallback, 
+			errorCallback, 
+			'Poynt', 
+			'launchBilling', 
+			[{
+				"paymentid": paymentid
+			}]);
+	},
+	launchAskConf: function (msg,successCallback, errorCallback){
+		cordova.exec(successCallback, 
+			errorCallback, 
+			'Poynt', 
+			'launchAskConf', 
+			[{"msg": msg}]);
+	},
+	launchSign: function (title,butt,msg,successCallback, errorCallback){
+		cordova.exec(successCallback, 
+			errorCallback, 
+			'Poynt', 
+			'launchSign', 
+			[{"msg": msg,"title":title,"leftbutton":butt}]);
+	},
+	launchInit: function (successCallback, errorCallback){
```

Comment:
Please change it to `init`. Launch implies that you are launching an activity or a view.